applications/nrf5340_audio/src/bluetooth/bt_management/scanning/bt_mgmt_scan_for_broadcast_internal.h

```diff
@@ -9,6 +9,13 @@
 
 #include <zephyr/bluetooth/bluetooth.h>
 
+/**
+ * @brief	Get the broadcast code.
+ *
+ * @param[in/out]	broadcast_code	Pointer to the broadcast code.
+ */
+void bt_mgmt_broadcast_code_get(uint8_t *broadcast_code);
```

Comment:
If the broadcast code is not set for some reason... Should this return an error?